### DISCUSSION TITLE
Create a place for official Adonis packages in the docs.adonisjs.com

### DISCUSSION THREAD

**Discussion by Barbapapazes** [adonisjs/core#3807] [Ideas]
Hello,

Does these packages must be added to the official documentation?

- https://github.com/adonisjs/route-model-binding
- https://github.com/adonisjs/lucid-slugify
- https://github.com/adonisjs/attachment-lite

It thing it could be very cool because they are so convenient!

**Comment by stereobooster** [comment]
https://packages.adonisjs.com/
